### ISSUE TITLE
[Bug]: Invariant Failed Error when trying to use getValues method

### ISSUE THREAD

**Issue by metzy23** [airjp73/remix-validated-form#148]
### Which packages are impacted?

- [X] `remix-validated-form`
- [ ] `@remix-validated-form/with-zod`
- [ ] `@remix-validated-form/with-yup`
- [ ] `zod-form-data`

### What version of these packages are you using?

- `remix-validated-form`: ^4.5.2
- `@remix-validated-form/with-yup`: ^2.0.1
- `yup`: ^0.32.11

### Please provide a link to a minimal reproduction of the issue.

https://stackblitz.com/edit/node-eruffr?file=app/routes/index.tsx

### Steps to Reproduce the Bug or Issue

1. Run the reproduction example. The error will show with no action needed
2. To see without error, comment out `const formValues = getValues();` (line 18 in index.tsx)

### Expected behavior

As a user, I expected to be able to see the current values of all the inputs in the form by calling `getValues` on the returned object from the `useFormContext` hook.

```js
const { getValues } = useFormContext('my-form');
const formValues = getValues();
```

### Screenshots or Videos

<img width="1105" alt="image" src="https://user-images.githubusercontent.com/15272328/181267901-bd9e8602-687d-46bd-a104-82f389c30039.png">

### Platform

- OS: macOS
- Browser: Chrome
- Version: 103.0.5060.114

**Comment by ThomiReynoso** [comment]
Hi! I had the same issue. I'm aware if it's resolved!

**Comment by danecando** [comment]
This is no doubt my bad. I'll check this out and work on a fix for it tonight

**Comment by metzy23** [comment]
Are there any updates on this issue?

**Comment by danecando** [comment]
Just opened a PR with a fix sorry gang

**Comment by metzy23** [comment]
No worries! Just wanted to check in. Thanks so much!!

**Comment by airjp73** [comment]
Fixed in v4.5.4
